### PR TITLE
NMA-6487: Add workout graph and statistics cards 

### PR DESCRIPTION
**Pull request by gurooerbech**
### What's new?
* Adds the workout graph and statistics cards that have been in the member app.
* Adds sample screen for the components.

### CODE REVIEW

File: sats-dna/src/main/kotlin/com/sats/dna/components/statistics/SatsWorkoutGraph.kt [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+                        .padding(top = 2.dp)
+                        .clip(SatsTheme.shapes.roundedCorners.small)
+                        .background(segmentColor),
+                )
+            }
+        }
+    }
+}
+
+data class GraphWeek(
```
Comment (gurooerbech) [inline sats-dna/src/main/kotlin/com/sats/dna/components/statistics/SatsWorkoutGraph.kt:180]: Any better name suggestions for this? 😝

File: sats-dna/src/main/kotlin/com/sats/dna/components/statistics/SatsWorkoutStatisticsCard.kt [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+class SatsWorkoutStatisticsCardItem private constructor(
+    val label: String,
+    val count: String,
+) {
+    constructor(label: String, count: Int) : this(label, "$count")
+    constructor(label: String, count: Double) : this(label, "$count")
+}
+
+@PreviewLightDark
+@Preview("2x font size", fontScale = 2f)
```
Comment (sindrenm) [inline sats-dna/src/main/kotlin/com/sats/dna/components/statistics/SatsWorkoutStatisticsCard.kt:116]: Maybe use `@PreviewFontScale` instead?

**Review by sindrenm — APPROVED** [review]
Sweet! 🎉

File: sats-dna/src/main/kotlin/com/sats/dna/components/statistics/SatsWorkoutGraph.kt (hunk 1 above)
Comment (sindrenm) [reply sats-dna/src/main/kotlin/com/sats/dna/components/statistics/SatsWorkoutGraph.kt:180]: `SatsWorkoutGraphCardWeek`?

**Comment by sindrenm** [comment]
Why is this breaking? 🤔

**Comment by gurooerbech** [comment]
> Why is this breaking? 🤔

Lol never mind, that was just me being eager with tags 🤓

**Pull request merged**